packages/langfuse-wrapper/scripts/setup_langfuse.sh

```diff
@@ -15,21 +15,41 @@
 #   ./setup_langfuse.sh
 #
 # Requirements:
-#   - Docker and Docker Compose
+#   - Docker 
 #   - curl
 #   - A .env.langfuse.local file in the env directory
 #
 # Note: This script is intended for local development use only.
 
 set -e
 
-SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SCRIPT_DIR=$(realpath "$(dirname "${BASH_SOURCE[0]}")")
 LANGFUSE_DOCKER_COMPOSE_URL="https://raw.githubusercontent.com/langfuse/langfuse/main/docker-compose.yml"
-LANGFUSE_DOCKER_COMPOSE_FILE="$SCRIPT_DIR/docker-compose.yaml"
+LANGFUSE_DOCKER_COMPOSE_FILE="docker-compose.yaml"
 LANGFUSE_ENV_FILE="$SCRIPT_DIR/../env/.env.langfuse.local"
 
+check_dependencies() {
+    local dependencies=("curl" "docker")
+    local missing_dependencies=()
+
+    for cmd in "${dependencies[@]}"; do
+        if ! command -v "$cmd" &> /dev/null; then
+            missing_dependencies+=("$cmd")
+        fi
+    done
+
+    if [ ${#missing_dependencies[@]} -ne 0 ]; then
+        echo "Missing dependencies: ${missing_dependencies[*]}"
+        echo "You can install them with: apt install -y ${missing_dependencies[*]}"
```

Comment:
ohh sorry this is on me - i wasn't clear just `${missing_dependencies[*]}` would be enough! we don't know what package managers people have